cmd/boulder-wfe2/main.go

```diff
@@ -334,8 +335,13 @@ func main() {
 		source := ratelimits.NewRedisSource(limiterRedis.Ring, clk, stats)
 		limiter, err = ratelimits.NewLimiter(clk, source, stats)
 		cmd.FailOnError(err, "Failed to create rate limiter")
-		txnBuilder, err = ratelimits.NewTransactionBuilderFromFiles(c.WFE.Limiter.Defaults, c.WFE.Limiter.Overrides)
+		if c.WFE.Features.OverridesFromDB {
```

Comment:
While this works, I think it should be `if features.Get().OverridesFromDB {` for the sake of consistency with all the other places we access feature flags. Same in ra/main.go.